projects/plugins/jetpack-boost/package-lock.json

```diff
@@ -0,0 +1,13392 @@
+{
```

Comment:
This should go away, since the monorepo uses pnpm. Also any scripts and stuff you have that run npm should be switched over.

Reply:
Removed https://github.com/Automattic/jetpack/pull/20512/commits/20530e155c219bc6baf3899c5469f03dabbe4025